esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -605,6 +610,7 @@ define(
     ZeroLogicalFilesCheckFilter: "Nincs a megadott feltételnek megfelelő adat! (A jó a szűrőfeltétel?)",
     Zip: "Zip",
     ZippedAnalysisPackage: "Tömörített elemzési csomag",
+    Zoom: "Nagyítás",
     Zoom100Pct: "Nagyitás 100%-ra",
```

Comment:
I mean not the new string is wrong. The next to it the Zoom100Pct should be "Nagyítás 100%-ra". The 'i' with different accent.